portal/src/it/scala/cromwell/pipeline/controller/UserControllerItTest.scala

```diff
@@ -23,22 +27,24 @@ class UserControllerItTest
   implicit val config: Config = TestContainersUtils.getConfigForPgContainer(container)
   private val components: ApplicationComponents = new ApplicationComponents()
 
+  import components.controllerModule.userController
+  import components.datastorageModule.userRepository
+
   override protected def beforeAll(): Unit =
     components.datastorageModule.pipelineDatabaseEngine.updateSchema()
 
-  import components.controllerModule.userController
-  import components.datastorageModule.userRepository
+  private val dummyUser: User = TestUserUtils.getDummyUser()
+  private val password: String = "-Pa$$w0rd-"
+  private val addUserFuture: Future[UserId] = userRepository.addUser(dummyUser)
```

Comment:
Change to `def`

Reply:
I removed this field because it casts duplicate errors by reason of adding the same user every time. As we don't have delete method for UserRepository I decided instantiate users in some cases.